src/main/java/ai/chat/service/UserRequestService.java

```diff
@@ -0,0 +1,25 @@
+package ai.chat.service;
+
+import ai.chat.model.AccountType;
+import ai.chat.model.User;
+import ai.chat.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import ai.chat.dto.*;
+
+@Service
+@AllArgsConstructor
+public class UserRequestService {
+    public UserRepository userRepository;
+
+    public UserResponseDto receiveMessage(String id) {
+        AccountType accountType = checkAccountType(id);
+        UserResponseDto userResponse = new UserResponseDto(id, accountType);
+        //if type is free pass to one model, if premium to another
+        return userResponse;
+    }
+    public AccountType checkAccountType(String id) {
```

Comment:
If this method is a check it is logical to return boolean, otherwise this is a simple getter, so no need to implement. Instead we can have isPremiumAccount() method that returns boolean.